feat: adiciona player para meditações

ThaisMap opened the pull request:
- adiciona página de  player para meditações
- confiigura navegação da lista de meditações para o player

Refs #177948288

ThaisMap committed:
feat: adiciona Player para meditações

- adiciona página PlayerMeditacao
- confiigura navegação da lista de meditações para o player

refs #177948288

kellynvd commented on src/pages/app/meditacoes/[id].tsx:
```diff
@@ -0,0 +1,223 @@
+    setIsPlaying(true)
+  }
+
+  const IconePlayPause = () => {
```
Aqui poderia ser algo assim pra não repetir o style (e _acho_ que também, não precisar do comentário do eslint nesse caso)

```
const CircleFilledIcon = isPlaying
      ? PauseCircleFilledIcon
      : PlayCircleFilledIcon

return <CircleFilledIcon style={{ fontSize: 90 }} />
```

ThaisMap replied:
Eu tinha tentado fazer assim, mas não acertei

kellynvd commented on src/pages/app/meditacoes/[id].tsx:
```diff
@@ -0,0 +1,223 @@
+    return (
+      <Container maxWidth="xs">
+        <LinkVoltar href="/app/meditacoes" />
+        <Box className={classes.titulo}>
```
Aqui acho que poderia ser só o loading ou o loading + o botão voltar

```
<Container maxWidth="xs">
  <LinkVoltar href="/app/meditacoes" />
  <Loading />
</Container>
)
```

kellynvd commented on src/pages/app/meditacoes/[id].tsx:
```diff
@@ -0,0 +1,223 @@
+    <Container maxWidth="xs">
+      <LinkVoltar href="/app/meditacoes" />
+      <Box className={classes.titulo}>
+        <Titulo>{meditacao ? meditacao.nome : 'loading'}</Titulo>
```
Aqui precisa dessa verificação, já que se tiver carregando vai cair no <EmptyPlayer /> ?

kellynvd commented:
Ficou com muito massa o player @ThaisMap :smiley: :clap:   

Só deixei uns comentários com algumas observações e tem mais dois pontos:

- [x] Acho que poderia mover a page meditacoes pra dentro da pasta meditacoes que você criou e renomear a página pra `index.tsx`
- [x] Ao clicar em voltar quando a meditação está tocando/rodando da esse erro:
![Screenshot from 2021-06-21 10 47 54](https://user-images.githubusercontent.com/13313889/122804235-05dd7f80-d29e-11eb-9d79-e2eb62bc493c.png)

ThaisMap committed:
fix: ajusta erro ao sair com áudio tocando

- Resolve erro o sair do player com áudio ainda tocando
- Remove testes e textos desnecessários
- Move página de listagem de meditações para dentro da pasta meditações

dodocinho committed:
Merge branch 'develop' into feat/player-meditacao
